build.yaml

```diff
@@ -69,6 +71,40 @@ release:
             ./build_${PACKAGE_TYPE}.sh
             find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \;
             pushd packages
+
+            # Install cpp-driver packages
+            sudo ${PACKAGE_INSTALL} cassandra*.${PACKAGE_TYPE}
+
+            # Create test file against cpp-driver and try to connect. Should compile and throw a connection error when executed
+            $CC -x c -o $TEST_PROGRAM - -lcassandra - <<EOF
+            #include <cassandra.h>
+            #include <stdio.h>
+
+            int main(int argc, char* argv[]) {
+            /* Setup and connect to cluster */
+            CassFuture* connect_future = NULL;
+            CassCluster* cluster = cass_cluster_new();
+            CassSession* session = cass_session_new();
+            /* Add contact points */
+            cass_cluster_set_contact_points(cluster, "127.0.0.1");
+            /* Provide the cluster object as configuration to connect the session */
+            connect_future = cass_session_connect(session, cluster);
+            cass_future_wait(connect_future);
+            printf("Success\n");
+            return 0;
+            }
+            EOF
+            
+            # Throw error if either compilation or test run fails
+            if [[ $? -ne 0 ]] ; then
+              exit 1
+            fi    
+            
+            if [[ $($TEST_PROGRAM) != *"Success"* ]] ; then
```

Comment:
* Why the different style bracketing for the conditional? Prefer consistency.

Reply:
Double brackets is a bash thing that give you more advanced functionality.

Reply:
I agree. We should be consistent though.

Reply:
Looks like the string matching i'm using requires double brackets :(